EIPS/eip-3102.md

```diff
@@ -0,0 +1,189 @@
+---
+eip: 3102
+title: Binary trie structure
+author: Guillaume Ballet (@gballet), Vitalik Buterin (@vbuterin)
+status: Draft
+discussions-to: https://ethresear.ch/t/binary-trie-format/7621
+status: Draft
+type: Standards Track
+category: Core
+created: 2020-09-01
+---
+
+## Simple Summary
+
+A proposal for changing the format of the data-storage structure from hexary to binary, merging the account and storage tries, and using the faster BLAKE2. The goal is to make stateless proofs more wieldy.
```

Comment:
Simple Summary should be akin to an email subject line, forum topic title, GitHub PR title, etc.  Also, goals should be part of the motivation.
```suggestion
Change storage structure from hexary to binary, merge the account and storage tries, and use BLAKE2.
```